sysl2/sysl/binexprEval.go

```diff
@@ -117,7 +119,8 @@ func (op DefaultBinExprStrategy) eval(txApp *sysl.Application, assign Scope, bin
 	panic(errors.Errorf("Unsupported operation:DefaultBinExprStrategy: %s", key))
 }
 
-func (op EvalLHSOverRHSStrategy) eval(txApp *sysl.Application, assign Scope, binexpr *sysl.Expr_BinExpr) *sysl.Value {
+func (op EvalLHSOverRHSStrategy) eval(txApp *sysl.Application,
+	assign syslutil.Scope, binexpr *sysl.Expr_BinExpr) *sysl.Value {
```

Comment:
Move `) *sysl.Value {` to its own line, adding a `,` to the end of this one.

Reply:
Done